Cargo.lock

```diff
@@ -386,11 +386,11 @@ dependencies = [
 
 [[package]]
 name = "expect-test"
-version = "1.0.2"
+version = "1.1.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "eab78d4c727fa879de40951ffc01ed781a231b28786b19d6bfbdc0d1ec92eb3e"
+checksum = "2300477aab3a378f2ca00a4fbd4dc713654ab7ed790e4017493cb33656280633"
 dependencies = [
- "difference",
+ "dissimilar",
```

Comment:
I wonder what keeps difference in tree?

Reply:
proc_macro_serve and test_utils have it in their Cargo.toml. Seems like only test_utils is actually using it

Reply:
If you want to I can send a patch to either:
 - use expect-test in test_utils
 - copy the formatting impl from expect-test to test_utils and use dissimilar there

Reply:
I think migrating to expect_test is better (but I didn't look at the code)